src/System.Data.Common/tests/System/Data/Common/RowUpdatedEventArgsTest.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Xunit;
+
+namespace System.Data.Common
+{
+    public class RowUpdatedEventArgsTest
+    {
+        [Fact]
+        public void ConstructorFail()
+        {
+            try
+            {
+                new RowUpdatedEventArgs(null, null, (StatementType)666, null);
+                Assert.False(true);
+            }
+            catch (ArgumentOutOfRangeException ex)
+            {
+                Assert.Equal(typeof(ArgumentOutOfRangeException), ex.GetType());
+                Assert.Null(ex.InnerException);
+                Assert.True(ex.Message.IndexOf(nameof(StatementType)) != -1);
+                Assert.True(ex.Message.IndexOf("666") != -1);
```

Comment:
I'd avoid using controversial values

Reply:
Oh, sorry! Didn't expect it to be a problem.
Just saw it in other test cases and adopted it. No big deal though, will change.